README.md

```diff
@@ -108,6 +108,50 @@ HTML styled "just right".
 Jekyll will print a local URL where the site can be viewed (usually,
 [http://0.0.0.0:4000/](http://0.0.0.0:4000/)).
 
+### Testing using Docker environment 
+
+#### Build environment
+A containerized development environment can be built using the local
+Dockerfile.
+
+Run the build-images.sh script to generate the development environment and
+associated images.
```

Comment:
```suggestion
A containerized development environment can be built using the local
Dockerfile. You can build it with the following command:

```

Reply:
Ah I missed that in my earlier removal. 
Thanks for the catch!